src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -134,13 +139,32 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
     };
   }, [relatedVenueEvents, userEventIds, selectedDayIndex, getEventLocation]);
 
+  const downloadCalenderPersonal = useCallback(() => {
```

Comment:
here and elsewhere a typo - should be  Calendar not Calender.
```
$> git grep -i calender
src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:  const downloadCalenderPersonal = useCallback(() => {
src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:      .map(prepareForSchedule(dayStart, userEventIds, "calender"))
src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:  const downloadCalenderAll = useCallback(() => {
src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:        <Button onClick={downloadCalenderPersonal}>
src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:        <Button onClick={downloadCalenderAll}>Download full schedule</Button>
src/components/organisms/NavBarSchedule/utils.ts:  if (target === "calender") {
src/utils/calendar.ts:  calendername: string
src/utils/calendar.ts:  const cal = ical({ name: calendername });
src/utils/calendar.ts:  const outputFile = `${calendername}_calendar.ics`;
```

smth like
```
git grep -l alender | xargs sed -i -e 's,alender,alendar,g'
```
should fix them all